crates/db_schema/src/utils.rs

```diff
@@ -59,6 +58,9 @@ pub const SITEMAP_LIMIT: i64 = 50000;
 pub const SITEMAP_DAYS: Option<TimeDelta> = TimeDelta::try_days(31);
 pub const RANK_DEFAULT: f64 = 0.0001;
 
+/// Change collapse limits from 8 to 11 so the query planner can find a better table join
+/// order for more complicated queries
+const CONNECTION_OPTIONS: [&str; 1] = ["geqo_threshold=12"];
```

Comment:
Comment says 11 but value is 12. Also this is only used once so you can mvoe the const inside the function.

Reply:
I removed the comment, but the geqo_threshold needed to be higher, 12 for some reason? I'll just remove the comment. I can't really figure out what these do or why they were added, so I just messed with it until the queries became fast again.

Reply:
afaik, they were added by @dullbananas  because the number of joins we have (had?) in the post_view / comment_view queries crossed this threshold, which hugely makes queries less efficient when the optimization that postgresql has to do to make the query efficient is to look at every column stat.

for example, as soon as you add a filter like "WHERE post_saved=true" to the post view query suddenly the query plan needs to look completely different, because now post_saved is the thing that filters out 99.9% of posts

Reply:
but to be clear I don't remember and didn't investigate these specific parameters and what they do, I just think these were added at the time we were discussing those things I mentioned